### ISSUE TITLE
Remove "sighted language" for Claim Cards

### ISSUE THREAD

**Issue by skylerschain** [department-of-veterans-affairs/va.gov-team#80626]
## Background Context

We've been instructed to remove "View details" since it's considered "sighted language" and replace it with simply "Details" on the claim card.

## Design

Before
<img width="462" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOUZUQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c0ccf617cded5d7b0bc87618c740f6ecf78a68cc/Screenshot%202024-04-11%20at%201.27.58%E2%80%AFPM.png" alt="Screenshot 2024-04-11 at 1.27.58 PM.png" />

After
<img width="472" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOUpUQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3f40078c90f3b8418fa0867b0639b99c1fb7fd56/Screenshot%202024-04-11%20at%201.28.03%E2%80%AFPM.png" alt="Screenshot 2024-04-11 at 1.28.03 PM.png" />

## Tasks
- [ ] Update all claim cards to no longer show 'View details' and to instead show 'Details'
- [ ] Update the aria-label on the claim cards to no longer show 'View details' and to instead show 'Details'

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Design performs design QA and verifies the implementation matches the design spec
- [ ] Accessibility specialist performs accessibility review (in code or design)
- [ ] Engineering identifies staging users required to test and shares account and credentials with design and product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria
- [ ] All claim cards to no longer show a 'View details' link and instead show a 'Details' link
- [ ] The aria-label on the claim card no longer shows 'View details' and instead shows 'Details'

**Comment by pmclaren19** [comment]
Discussed with @skyler and @davidakennedy via slack.
- We will update all claim cards to no longer show 'View details' and to instead show 'Details'
- We will update the aria-label to no longer show 'View details' and to instead show 'Details'
I will update the ticket details to reflect this and add ac.

**Comment by pmclaren19** [comment]
Verified changes in Staging and looks good!
